LICENSE

```diff
@@ -0,0 +1,29 @@
+Copyright (c) 2021, PlanetScale Inc. All rights reserved.
+Copyright (c) 2013, The GoGo Authors. All rights reserved.
+Copyright (c) 2018, The Go Authors. All rights reserved.
```

Comment:
Should we add `Cosmos SDK Team` here?

Reply:
I have put down the two teams that worked on it (regen and aib) but if cosmos sdk team is preferred or more correct i will change it